src/Console/Command/DeserializeCommand.php

```diff
@@ -27,7 +27,7 @@ protected function configure()
         $this
             ->setName('deserialize')
             ->setDescription('Benchmarks the deserialization process')
-            ->addOption('interactions', 'i', InputOption::VALUE_REQUIRED, 'Amount of deserialization each vendor will perform', 100)
+            ->addOption('samples', 's', InputOption::VALUE_REQUIRED, 'Amount of deserialization each vendor will perform', 100)
```

Comment:
Change command's description from `Amount of deserialization each vendor will perform` to `Amount of samples the application will perform`